src/Randomizer.Data/Configuration/Yaml/Sassy/hint_tiles.yml

```diff
@@ -0,0 +1,31 @@
+﻿RequestedHintTile:
+  - Text: 'If you were to look at that hint tile, it would say {0}'
+  - Text: 'I guess you''re too lazy to go there yourself? Fine. It says {0}'
+  - Text: 'Can''t read it yourself? It says {0}'
+  - Text: 'From what I''ve heard, it says {0}'
```

Comment:
This would be the perfect opportunity to put in something like "It says Shaktool is required", followed by a pause and then the actual hint text.